wiki/Game_modifier/Nightcore/id.md

```diff
@@ -0,0 +1,48 @@
+---
+stub: true
+tags:
+  - nightcore
+  - NC
+  - mod
+  - game modifier
+---
+
+# Nightcore (mod)
+
+![Ikon mod NC](/wiki/shared/mods/NC.png "Ikon mod Nightcore (NC)")
+
+*Berbeda konteks dengan [Nightcore (genre)](https://en.wikipedia.org/wiki/Nightcore "Wikipedia").*\
```

Comment:
```suggestion
*Berbeda konteks dengan [Nightcore (aliran)](https://en.wikipedia.org/wiki/Nightcore "Wikipedia").*\
```

Reply:
should we replace it instead, similar to the beatmap listing site?

Reply:
i don't think so...